spec/acceptance/exec_powershell_spec.rb

```diff
@@ -561,6 +591,7 @@ def platform_string(host, windows, posix)
       $pr = New-Object Security.Principal.WindowsPrincipal $id
       if(!($pr.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator))){Write-Error "Not in admin"}
     PS1
+    #require 'pry'; binding.pry
```

Comment:
One line to remove

Reply:
I should really have a pre push hook for this...